{"name": "test_authenticated_with_bad_hesh", "status": "broken", "statusDetails": {"message": "KeyError: 'accessToken'", "trace": "self = <test_create_order.TestCreateOrder object at 0x000001CFA9D034A0>, data_new3 = (<Response [403]>, {'email': 'ronald29@example.org', 'name': 'Debra', 'password': 'bAj2Sb$ky@'})\n\n    @allure.description('Создание заказа c авторизаией и невалиднным хешем')\n    def test_authenticated_with_bad_hesh(self, data_new3):\n        response_create_user, payload = data_new3\n>       token = response_create_user.json()[\"accessToken\"]\nE       KeyError: 'accessToken'\n\ntests\\test_create_order.py:51: KeyError"}, "description": "Создание заказа c авторизаией и невалиднным хешем", "start": 1719430161098, "stop": 1719430161099, "uuid": "492a111e-da7c-4b2e-84e9-74c088cdbab3", "historyId": "b1623cf3ac74887bc51df7335e814887", "testCaseId": "b1623cf3ac74887bc51df7335e814887", "fullName": "tests.test_create_order.TestCreateOrder#test_authenticated_with_bad_hesh", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "DESKTOP-4LHP98R"}, {"name": "thread", "value": "24376-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}